AppInspector/Resources/defaultRules.json

```diff
@@ -0,0 +1,17075 @@
+[
```

Comment:
Delete this entire file, it isn't needed as it'll get generated dynamically.  You can add `Resources/defaultRules.json` to the `.gitignore` to make sure no one checks it in by mistake.

Reply:
Yep, I see.